src/style.css

```
@@ -0,0 +1,54 @@
+body {
+  margin: 0;
+  --block-height: 200px;
+}
+
+#header,
+#footer,
+.important,
+.interesting,
+.surprising {
+  width: 300px;
+  height: 100px;
+  margin: 0 auto;
+}
+
+#header {
+  background-color: blue;
+}
+
+#header:hover {
+  height: var(--block-height);
+}
+
+#footer {
+  background-color: black;
+}
+
+#footer:hover {
+  height: var(--block-height);
+}
+
+.important {
+  background-color: red;
+}
+
+.important:hover {
+  height: var(--block-height);
+}
```

```suggestion
.important:hover
.surprising:hover 
#footer:hover {
  height: var(--block-height);
}
```
сделай вот так где это нужно

---

🤦‍♀️